packages/runtime/src/runtimes/react/index.tsx

```diff
@@ -158,6 +158,10 @@ export function useIsInBuilder(): boolean {
   return useSelector(state => ReactPage.getIsInBuilder(state))
```

Comment:
I don't totally understand what this class represents. We should talk through it in pairing today

Reply:
Which class? If you mean `ReactPage`, that's not a class, it's just a module that has a bunch of named exports and I'm importing with that name like so:

```typescript
import * as ReactPage from './path/to/react-page.ts
```